src/elements/components/model-card.module.scss

```
@@ -48,6 +53,17 @@
         }
     }
 
+    .paired {
+        position: absolute;
+
+        backdrop-filter: blur(2rem) saturate(1.5) opacity(0.95);
+
+        @include themed using($t) {
+            background: linear-gradient(transparent, t($t, white, $fade-900))
+                t($t, color.change(white, $alpha: 0.7), color.change($fade-900, $alpha: 0.7));
```

I think we give more opacity to the dark background. Bright color shine through a lot more than in light mode.
```suggestion
                t($t, color.change(white, $alpha: 0.7), color.change($fade-900, $alpha: 0.8));
```